pynws/nws.py

```diff
@@ -24,47 +27,64 @@ def __init__(self, message):
 class Nws:
     """Class to more easily get data for one location."""
 
-    def __init__(self, session, userid, latlon=None, station=None):
```

Comment:
I changed it to be required, but that might not be right. How did this work if `latlon` was optional?

Reply:
If you only wanted to get the observations from a specific station, you don't need `latlon`.

Reply:
I need to work on better doc strings, which would have helped here.  It isn't very obvious.

Reply:
> If you only wanted to get the observations from a specific station, you don't need `latlon`.

I restored that behavior.

Reply:
> I need to work on better doc strings, which would have helped here. It isn't very obvious.

FWIW, I'm using a VS Code extension called autoDocstring to format the doc strings.